test/integration/BUILD

```diff
@@ -230,6 +230,7 @@ exports_files(["test_utility.sh"])
 
 envoy_sh_test(
     name = "hotrestart_test",
+    size = "enormous",
```

Comment:
We need to fix envoy_sh_test to actual pass this size flag to envoy_cc_test. I'll put up a PR that does both, since I think @jmarantz may already be offline.

Reply:
See #11478

Reply:
I'll close this in favor of yours.